maths/factorial_iterative.py

```diff
@@ -2,11 +2,13 @@
 
 
 n=int(input("Enter a number"))
-
 def factorial(n):
     fact=1
```

Comment:
The function should raise an ValueError just like __math.factorial()__ does:

% __python3__
```
>>> import math
>>> math.factorial(-1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: factorial() not defined for negative values
```